### PR TITLE
hw-005b-solution

### PR DESCRIPTION
**Pull request by SviatoslavExpert**

### CODE REVIEW

File: src/main/java/core/basesyntax/BinaryString.java [hunk 1]
```diff
@@ -7,6 +7,16 @@
...
     public String toBinaryString(int value) {
-        return null;
+        int number;
+        String s = "";
+        if (value == 0) {
+            s = "0";
+        }
+        while (value > 0) {
+            number = value % 2;
+            s = "" + number + s;
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/BinaryString.java]: we can write:
```suggestion
            s = "" + value % 2 + s;
```

Comment (SviatoslavExpert) [reply src/main/java/core/basesyntax/BinaryString.java]: Fixed. Thank you.

File: src/main/java/core/basesyntax/BinaryString.java [hunk 2]
```diff
@@ -7,6 +7,14 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        String s = "";
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/BinaryString.java]: try to use full name of the variable

Comment (SviatoslavExpert) [reply src/main/java/core/basesyntax/BinaryString.java]: Full name is used.  Now it's binaryString instead of s. 
Done!

**Pull request closed without merging**